### ISSUE TITLE
V1.5.0 RC1: LUA disrespects attribute zero digits after decimal point

### ISSUE THREAD

**Issue by digitalonkel** [FrSkyRC/ETHOS-Feedback-Community#3375]
Seems the %.0 is not respected, see photo (e.g. RSSI or EGT, should be 0 digits after decimal point)
worked in v1.4.x
![IMG_0137](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/62343908/cd5bbbbf-920e-4771-9292-50921913ba4c)

**Comment by bsongis-frsky** [comment]
What is the Lua function you are calling to do that?

**Comment by bsongis-frsky** [comment]
Your code converts a float to string, then again to float. What about this:

```lua
lcd.drawNumber(305, 22, RSSI1Value, UNIT_DB, 0)
```

**Comment by digitalonkel** [comment]
Same effect with your code. I get 6 digits after dec. point. Old code and your code work at versions 1.4.x, so it is a bug in V1.5

**Comment by digitalonkel** [comment]
Lior Z. and me did some additional tests. Adding a unit, will have the effect, that 0 digits are respected.
![e1dd5389-3415-40b8-b186-a88c812db0be](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/62343908/95a06693-2f7c-44cd-8603-a6c93d8730c9)
w/o the attribute UNIT_RPM it will be shown x.000000
definately a bug, which was not there in v1.4

**Comment by digitalonkel** [comment]
it was not visible in the simulator

**Comment by bsongis-frsky** [comment]
> Adding a unit, will have the effect, that 0 digits are respected

It is exactly what you want to have (title of the issue), or I miss something?

**Comment by digitalonkel** [comment]
> > Adding a unit, will have the effect, that 0 digits are respected
> 
> It is exactly what you want to have (title of the issue), or I miss something?

Exactly. For my requirements I found a solution. Nevertheless it is an issue that a default structure of x.xxxxxx is shown although digits are defined.

**Comment by digitalonkel** [comment]
Confirmed. It is working with the nighly from feb. 8th

**Comment by i3dm** [comment]
unfortunately it seems that this fix from the nightly is not implemented in1.5.0 release, as the release shows:
lcd.drawNumber(640, 285, PumpValue*A4Mult, 0)
with a value of 0 as 0.000000

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/8968780/4fe55979-f3ac-4576-9841-d572642f0037)


**this does NOT occur on 1.5.1 sim, only occured on the X20S 1.5.0 itself.**
could be that this was only fixed in 1.5.1?
@bsongis-frsky please reopen the issue if needed.
